test/setup/babel-register.js

```diff
@@ -0,0 +1,6 @@
+require('babel-core/register')({
+    presets: [
+        'es2015',
+        'react',
+    ],
+});
```

Comment:
This is a really interesting approach to providing the `presets` config to tests, and yet avoiding it in the "global" root `.babelrc`. Why wasn't repo-global config possible?